### DISCUSSION TITLE
Detect when apis returns a 429

### DISCUSSION THREAD

**Discussion by danielpetrica** [dedoc/scramble#428] [Ideas]
It would be nice if the library could detect when the code returns a response with status code 429 from inside the controlle. iI need to use this to check if the user created an element of a model in the last 30 minutes already and in theat case return him a 429 status code adn message. I found that scramble doesn't detect it. here's how I return it 
`return response()->json('error message', 429 );`
If you detect other status codes this should not be hard to detect.

**Comment by romalytvynenko** [comment]
@danielpetrica I'm pretty sure it already works
